### PR TITLE
LRS dropdown

### LINKED ISSUE #914: Add LRS url to settings screen
For the ARETE pilot, we need the LRS to be configurable - please add to settings screen as a dropdown, so the API key is hidden from the user.

### PR DESCRIPTION
**Pull request by robhillman97**
Added drop down to settings menu's to allow users to switch between WEKIT and ARETE LRS

closes #914

### CODE REVIEW

File: Assets/MirageXR/Player/Scripts/Mobile/Popup/SettingsView.cs [hunk 1]
```diff
@@ -10,25 +10,24 @@ public class SettingsView : PopupBase
     private const string VERSION_FORMAT = "Version {0}";
     
     [SerializeField] private TMP_Text _txtVersion;
-    [SerializeField] private ExtendedInputField _inputFieldRecordStore;
     [SerializeField] private ExtendedInputField _inputFieldMoodleAddress;
     [SerializeField] private Toggle _togglePublicUpload;
     [SerializeField] private Toggle _toggleUiForKids;
     [SerializeField] private Button _btnSave;
     [SerializeField] private Button _btnReset;
+    [SerializeField] private Dropdown _LRSDropdown;
```
Comment (cawke) [inline Assets/MirageXR/Player/Scripts/Mobile/Popup/SettingsView.cs]: fix the name

File: Assets/MirageXR/Player/Scripts/Managers/EventManager.cs [hunk 2]
```diff
@@ -817,5 +817,13 @@ public static void NotifyOnMobileAddStepContentPressed()
...
             MobileAddStepContentPressed?.Invoke();
         }
+
+
+        public delegate void xAPIChangedDelegate(int option);
+        public static event xAPIChangedDelegate XAPIChanged;
+        public static void NotifyxAPIChanged(int option)
+        {
+            XAPIChanged?.Invoke(option);
+        }
```
Comment (cawke) [inline Assets/MirageXR/Player/Scripts/Managers/EventManager.cs:827]: in the future we need to get rid of event creation. For now, let it be

**Review by cawke — CHANGES_REQUESTED** [review]
Add screenshots of what the menu now looks like and screenshots of what you have changed in the prefabs

**Comment by robhillman97** [comment]
> 

This has currently only been added to the HoloLens and old mobile UI. I am looking at how to add this to the new UI at the moment. 

LRS Dropdown

LRS Dropdown HL

**Review by cawke — APPROVED** [review]
Thanks

**Pull request merged**